test/helpers/deployer.js

```diff
@@ -104,7 +104,10 @@ class Deployer {
       registry: this.registry,
       rootChain: this.rootChain,
       stakeManager: this.stakeManager,
-      stakeToken: this.stakeToken
+      stakeToken: this.stakeToken,
+      governance: this.governance,
+      proxy: proxy,
```

Comment:
How about giving it a more descriptive name?

Reply:
It follows project's terms and contract names. Any suggestions?

Reply:
I renamed proxy -> stakeManagerProxy to be more clear what kind of proxy is that.